web/src/styles.css

```diff
@@ -2305,4 +2305,77 @@ div::-webkit-scrollbar-track-piece {
 div::-webkit-scrollbar-thumb {
     background: lightgray;
     border-radius: 3px;
+  
```

Comment:
requires a "}". I made a mistake when merging the master branch.

Reply:
@ovbm if this was a mistake on your side, can you fix it?